setup.sh

```diff
@@ -457,11 +488,113 @@ startKuzzle() {
   done
 }
 
-isKuzzleRunning() {
-  write "[ℹ] Checking that everything is running"
+installSystemService() {
+  KUZZLE_ETC_PATH="/etc/kuzzle"
```

Comment:
`/etc` is traditionally used to store configuration.

A docker-compose file is more a similar to a launcher to me (i.e. *.desktop files).
Can we use a more generic location? (I suggest `/opt/kuzzle`)